src/js/components/TableCell/doc.js

```diff
@@ -42,3 +42,91 @@ export const doc = TableCell => {
 
   return DocumentedTableCell;
 };
+
+export const themeDoc = {
+  'table.body.align': {
```

Comment:
they should go in tableBody no?

Reply:
I can't see where this is being used in the code.